controls/raddiagram/features/services.md

```diff
@@ -86,6 +86,8 @@ In order to manipulate, change or access a __Diagramming Service__, you have to
 		End Class	
 	{{endregion}}
 
+	> When registering a new [SerializationService](#serializationservice), you also have to re-initialize the [DataTransferService](#datatransferservice).

```

Comment:
How to re-initalize the DataTransferService? Give a code example. Or provide description on the topic.